src/components/cap-reporter.js

```diff
@@ -51,15 +56,15 @@ export default class CapReporter extends LitElement {
 				text-decoration: underline;
 			}
 
-			.reporters__main {
+			.reporter {
```

Comment:
Renamed this for consistency with BEM (which stands for "Block Element Modifier") 

This is what would be considered a block, rather than an element. A `.block` is the primary, or parent, component. BEM class names always start with the name of the Block. An example of a block might be a site's navigation, hero section, or footer. Any container that holds other code could be a Block.

You can compare that to an element:

A .block__element is an Element inside a Block, or inside the parent component. It is named using two underscores between the Block and Element to denote that the Element is a child, or descendent, of the Block. An Element could be a title or a link. It's anything inside the parent Block. 

BEM isn't intended to communicate hierarchy, so if a block contains another block, you don't need to indicate that in the name. 

For example these could both be blocks, and aren't named in way that communicates that one contains another: 

`article`
`image`

If an image in an article contained a caption, that caption might be named `image__caption` not `article__image__caption`